daprdocs/content/en/developing-applications/debugging/bridge-to-kubernetes.md

```diff
@@ -0,0 +1,26 @@
+---
+type: docs
+title: "Bridge to Kubernetes support for Dapr services"
+linkTitle: "Bridge to Kubernets"
+weight: 300
+description: "Debug Dapr apps locally which still connected to your Kubernetes cluster"
+---
+
+Bridge to Kubernetes allows you to run and debug code on your development computer, while still connected to your Kubernetes cluster with the rest of your application or services. This type of debugging is often called *local tunnel debugging*. 
+
+{{< button text="Learn more about Bridge to Kubernetes" link="https://code.visualstudio.com/docs/containers/bridge-to-kubernetes" >}}
+
+## Debug Dapr apps
+
+Bridge to Kubernetes supports debugging Dapr apps on your machine, while still having them interact with the services and applications running on your Kubernetes cluster. This example showcases Bridge to Kubernetes enabling a developer to debug the [distributed calculator quickstart](https://github.com/dapr/quickstarts/tree/master/distributed-calculator):
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/rxwg-__otso" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
One thing that is missing in the video is that, to use Bridge, you need to have a debug configuration to run your project locally. I created a default Node.js profile for the demo, and mentioned that in the call, but to help the customers it might make sense to add it to the sample app directly.
I'm happy to send a PR to update the sample app if it helps. Also, this is probably related to my other comment about providing step by step instructions.